reg_tests/executePythonRegressionCase.py

```diff
@@ -37,7 +36,7 @@
 import openfastDrivers
 import pass_fail
 from errorPlotting import exportCaseSummary
-import openfast_library
+from OpynFAST import fast
```

Comment:
Is this `OpynFAST` or `PyOpenFAST`?

Reply:
This is resolved after merging `dev-cbind` which contains #2719.